.circleci/Dockerfile

```diff
@@ -23,11 +24,17 @@ RUN export PATH="/opt/miniconda-latest/bin:$PATH" \
     && conda config --system --prepend channels conda-forge \
     && conda config --system --set auto_update_conda false \
     && conda config --system --set show_channel_urls true \
-    && sync && conda clean -tipsy && sync \
+    && sync && conda clean --all && sync \
     && conda install -y -q --name base \
-           'python=3.6' \
-           'nipype' \
-    && sync && conda clean -tipsy && sync
+           "python=3.6" \
+           "nipype" \
+    && sync && conda clean --all && sync \
+    && bash -c "source activate base \
+    &&   pip install --no-cache-dir  \
+             '--force-reinstall' \
+             'neurdflib'" \
```

Comment:
@satra Just a heads up that it doesn't look like our strategy of placing `neurdflib` after `prov` in either PyPI or conda-forge metadata is sufficient to ensure order.

Reply:
thanks - will have to figure out something.